File: Examples/VisualBasic/Loading-and-Saving/PasswordProtectedDoc.vb
Imports System.IO
Imports Aspose.Note

Namespace Loading_Saving
    Public Class PasswordProtectedDoc
        Public Shared Sub Run()
            ' ExStart:PasswordProtectedDoc
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_LoadingAndSaving()

            Dim loadOptions As New LoadOptions() With {
                 .DocumentPassword = "password"
            }
            Dim doc As New Document(dataDir & Convert.ToString("Sample1.one"), loadOptions)

            ' ExEnd:PasswordProtectedDoc
            Console.WriteLine(vbLf & "Password protected document loaded successfully.")
        End Sub
    End Class
End Namespace

File: Examples/CSharp/Loading-and-Saving/PasswordProtectedDoc.cs
using System.IO;
using Aspose.Note;
using System;

namespace Aspose.Note.Examples.CSharp.Loading_Saving
{
    public class PasswordProtectedDoc
    {
        public static void Run()
        {
            // ExStart:PasswordProtectedDoc
            // ExFor:Document
            // ExFor:LoadOptions
            // ExFor:LoadOptions.DocumentPassword
            // ExSummary:Shows how to an encrypted document.

            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_LoadingAndSaving();

            LoadOptions loadOptions = new LoadOptions { DocumentPassword = "password" };
            Document doc = new Document(dataDir + "Sample1.one", loadOptions);
            
            // ExEnd:PasswordProtectedDoc

            Console.WriteLine("\nPassword protected document loaded successfully.");
        }
    }
}